get_github_coderabbit_review/coderabbit_fetcher/ai/sentiment_analyzer.py

```
@@ -0,0 +1,704 @@
+                EventType.PROCESSING_COMPLETED,
+                source="SentimentAnalyzer",
+                data={
+                    "method": "ai",
+                    "sentiment_score": result.sentiment_score,
+                    "sentiment_label": result.sentiment_label.value,
+                    "confidence": result.confidence,
+                    "response_time_ms": response.response_time_ms,
+                },
+            )
+
+            return result
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)